src/lib/components/ui/ScrollView/ScrollView.jsx

```diff
@@ -214,6 +290,13 @@ ScrollView.propTypes = {
   arrows: PropTypes.bool,
   arrowsColor: PropTypes.string,
   arrowsScrollStep: PropTypes.number,
+  /**
+   * Auto scroll mechanism requires to have `key` prop set for every child present in `children`
+   * prop because it detects changes of these keys. Otherwise it will not work.
+   *
+   * See https://reactjs.org/docs/lists-and-keys.html#keys
+   */
+  autoScroll: PropTypes.oneOf(['always', 'detectEnd', 'off']),
```

Comment:
What is the difference between `always` and `detectEnd`? Please include better description.

Reply:
Will update documentation of it.

always = It always autoscroll to the end of the viewport
detectEnd = It autoscroll to the end of the viewport only when end of the viewport is detected when content is changed (you need to be scrolled at the bottom/right [depend on direction] to allow autoscroll functionality)